jbpm-services/jbpm-services-ejb/jbpm-services-ejb-timer/src/main/java/org/jbpm/services/ejb/timer/EJBTimerScheduler.java

```diff
@@ -139,11 +146,12 @@ public boolean removeJob(JobHandle jobHandle) {
 	}
 	
 	public TimerJobInstance getTimerByName(String jobName) {
-	    
-	    if (localCache.containsKey(jobName)) {
-	        logger.debug("Found job {} in cache returning", jobName);
-	        return localCache.get(jobName);
-	    }
+    	if (USE_LOCAL_CACHE) {
+    		if (localCache.containsKey(jobName)) {
```

Comment:
Just noticed that this can be made easier to read by ```if (USE_LOCAL_CACHE && localCache.containsKey(jobName))```. But I am fine with either way...